### PR TITLE
Add support for Scala 2.11

### PR DESCRIPTION
**Pull request by mijicd**
### Summary

- added support for 2.11
- enabled snapshot releases

### CODE REVIEW

**Comment by mijicd** [comment]
cc @jeremyrsmith @toxicafunk

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,7 +1,10 @@
 version: 2.1
 
+scala_211: &scala_211
+  SCALA_VERSION: 2.11.12
+
 scala_212: &scala_212
-  SCALA_VERSION: 2.12.8
+  SCALA_VERSION: 2.12.10
```
Comment (softinio) [inline .circleci/config.yml]:
```suggestion
  SCALA_VERSION: 2.12.11
```

**Comment by mijicd** [comment]
@softinio silencer doesn't like the suggested change :)

**Comment by softinio** [comment]
> @softinio silencer doesn't like the suggested change :). Reverting it for now.

Good to know thanks :-)

**Pull request merged**